test/modules/Math/signbit-test.chpl

```diff
@@ -0,0 +1,16 @@
+use Math;
```

Comment:
You do not need to explicitly include `Math`, it is an implicitly included module. 

Also, `test/library/standard/Math/` is probably a more appropriate place for this test.